File: src/my_project/easy_problems/from101to150/check_if_sentence_is_pangram.py
class Solution:
    def checkIfPangram(self, sentence: str) -> bool:
        answer = True
        letters = "abcdefghijklmnopqrstuvwxyz"

        for letter in letters: 
            if letter not in sentence:
                answer = False 
        

        return answer


solution = Solution()

print(solution.checkIfPangram(sentence = "thequickbrownfoxjumpsoverthelazydog"))

File: my_project_typescript/easy_problems_typescript/from1to50/checkIfSentenceIsPangram.ts
function checkIfPangram(sentence: string): boolean {
    let answer: boolean = true 
    let letters: string = "abcdefghijklmnopqrstuvwxyz"

    for (let letter of letters){
        if (!sentence.includes(letter)){
            answer = false
        }
    }
    return answer 

};

console.log(checkIfPangram("thequickbrownfoxjumpsoverthelazydog"))

console.log(checkIfPangram("leetcode"))
